src/software/jetson_nano/services/network.cpp

```diff
@@ -30,12 +30,33 @@ std::tuple<TbotsProto::PrimitiveSet, TbotsProto::World> NetworkService::poll(
                                                                    world_msg};
 }
 
+float calculatePacketLossRate(std::queue<uint64_t> recent_proto_seq_nums,
+                              uint64_t seq_num)
+{
+    recent_proto_seq_nums.push(seq_num);
```

Comment:
```suggestion
float calculatePacketLossRate(const std::queue<uint64_t>& recent_proto_seq_nums,
                              uint64_t seq_num)
{
    recent_proto_seq_nums.push(seq_num);
```
The queue is being copied right now, so the actual queue is never updated right now.

Also, `push`ing the new sequence number in this function doesn't really match what the function is intended to do (calculate packet loss rate), so I think we should have the caller `push` the sequence number before calling it.